CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## [v1.1.1] - 2024-09-16
+### Changed
+
+- [[TRT-558]](https://bugs.earthdata.nasa.gov/browse/TRT-558)]
```

Comment:
```suggestion
- [[TRT-558](https://bugs.earthdata.nasa.gov/browse/TRT-558)]
```

Reply:
Thanks for catching that: [2cd831](https://github.com/nasa/harmony-swath-projector/pull/18/commits/2cd831067567814c0997e6c4bfb9820cff502d67).